lib/explorer/query.ex

```diff
@@ -320,7 +320,7 @@ defmodule Explorer.Query do
     then: 2
   ]
 
-  @kernel_only kernel_only -- kernel_only -- kernel_all
+  @kernel_only kernel_only -- (kernel_only -- kernel_all)
```

Comment:
Going to revert this only for the CI. We can update back when Elixir 1.18 is adopted as default version.

```suggestion
  @kernel_only kernel_only -- kernel_only -- kernel_all
```